### PR TITLE
Rke2 monitoring

### PR DESCRIPTION
**Pull request by dragonleman**

### CODE REVIEW

**Review by domq — APPROVED** [review]

File: ansible/roles/rke2-cluster/tasks/wpn-mariadb.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@
           enabled: true
 
 
-- name: Create mariadb/deployment
+- name: Create mariadb/databse
```
Comment (ponsfrilus) [inline ansible/roles/rke2-cluster/tasks/wpn-mariadb.yml:21]: database

File: ansible/roles/rke2-cluster/tasks/main.yml [hunk 2]
```diff
@@ -56,22 +68,22 @@
...
+- name: "Deploy latest version of mariadb-operator-crds chart inside {{ mariadb_namespace_name }} namespace (and create it)"
   kubernetes.core.helm:
     name: mariadb-operator-crds
     chart_ref: mariadb-operator/mariadb-operator-crds
     release_namespace: "{{ mariadb_namespace_name }}"
   run_once: true
 
 
-- name: "Deploy latest version of mariad-operator chart inside {{ mariadb_namespace_name }} namespace (and create it)"
+- name: "Deploy latest version of mariadb-operator chart inside {{ mariadb_namespace_name }} namespace (and create it)"
```
Comment (ponsfrilus) [inline ansible/roles/rke2-cluster/tasks/main.yml:86]: MariaDB?

**Comment by domq** [comment]
@dragonleman @dragon_leman : 👌 mergé

File: ansible/roles/rke2-cluster/tasks/main.yml [hunk 3]
```diff
@@ -56,22 +68,22 @@
...
-- name: Add helm repo for mariad-operator
+- name: Add helm repo for mariadb-operator
   kubernetes.core.helm_repository:
     name: mariadb-operator
     repo_url: "https://helm.mariadb.com/mariadb-operator"
   run_once: true
 
 
-- name: "Deploy latest version of mariad-operator-crds chart inside {{ mariadb_namespace_name }} namespace (and create it)"
+- name: "Deploy latest version of mariadb-operator-crds chart inside {{ mariadb_namespace_name }} namespace (and create it)"
```
Comment (ponsfrilus) [inline ansible/roles/rke2-cluster/tasks/main.yml:78]: MariaDB?

File: ansible/roles/rke2-cluster/tasks/main.yml [hunk 4]
```diff
@@ -23,6 +23,18 @@
 # créer le robot account avec le secret
 
 
+- name: "Create {{ monitoring_namespace_name }} namespace if not exist"
```
Comment (ponsfrilus) [inline ansible/roles/rke2-cluster/tasks/main.yml:26]: exists?

**Review by ponsfrilus — CHANGES_REQUESTED** [review]
Minor typos

**Pull request merged**